.ado/jobs/desktop.yml

```diff
@@ -66,6 +66,17 @@ jobs:
         clean: false
         submodules: false
 
+      # Testing to see if syntax is corerct so we don't have ot wait for 
```

Comment:
IF this is successfull inthe PR check, I'll remove it as we only need it in the Publish pipeline...